src/include/catalog/zone_map_catalog.h

```diff
@@ -64,11 +64,9 @@ class ZoneMapCatalog : public AbstractCatalog {
     TYPE = 6
   };
 
-  enum class ZoneMapOffset { 
-    MINIMUM_OFF = 0, 
-    MAXIMUM_OFF = 1, 
-    TYPE_OFF = 2 
-  };
+  enum ZoneMapOffset { MINIMUM_OFF = 0, MAXIMUM_OFF = 1, TYPE_OFF = 2 };
+
```

Comment:
I'm not sure why a strongly typed enum has been replaced by an old-style enum. It's better to stick to stongly typed enums. 

The same applies to a bunch of places where old-style enums have been used.